src/js/sign.js

```diff
@@ -0,0 +1,221 @@
+const emailDiv = document.querySelector('.content1_div2_div1'); //이메일 상위 div
+const pwDiv = document.querySelector('.content1_div2_div2_div-input'); //비밀번호 상위 div
+const emailInput = document.querySelector('.content1_div2_div1_input'); //이메일 input
+const pwInput = document.querySelector('.content1_div2_div2_input'); //비밀번호 input
+const pwSignupDiv1 = document.querySelector('.signup_pw1'); //signup 비밀번호 상위 div1
+const pwSignupDiv2 = document.querySelector('.signup_pw2'); //signup 비밀번호 상위 div2
+const pwSignupInput1 = document.querySelector('.signup_pw_input1'); //signup 비밀번호 input1
+const pwSignupInput2 = document.querySelector('.signup_pw_input2'); //signup 비밀번호 input2
+const eye = document.querySelector('.content1_div2_div2_img'); //signin 눈
+const eye_signup1 = document.querySelector('.content1_div2_div2_img_signup1'); //signup 눈1
+const eye_signup2 = document.querySelector('.content1_div2_div2_img_signup2'); //signup 눈2
+const login = document.querySelector('.content1_div3_div'); //로그인,회원가입 버튼
+
+const VERIFYEAMIL = 'test@codeit.com';
+const VERIFYPW = 'codeit101';
```

Comment:
상수는 대문자로 하신 거 너무 좋은 것 같아요
일반적으로 언더바 (_) 로 단어를 구분하는 편이에요!

```suggestion
const VERIFY_EAMIL = 'test@codeit.com';
const VERIFY_PW = 'codeit101';
```

Reply:
오옹 대문자만 기억나서 했었는데 다음번에 바꾸겠습니다!

Reply:
BEST